app/javascript/packages/eslint-plugin/configs/recommended.js

```diff
@@ -21,6 +21,7 @@ const config = {
     'max-len': 'off',
     'max-classes-per-file': 'off',
     'newline-per-chained-call': 'off',
+    'no-cond-assign': 'off',
```

Comment:
I really like this rule, it helps catch silly mistakes. What about `except-parens`, which is what it looks like you used?

```
  while ((sibling = sibling.nextElementSibling)) {
```

Reply:
> I really like this rule, it helps catch silly mistakes. What about `except-parens`, which is what it looks like you used?

Yeah, I wasn't feeling 100% about this change, but I really didn't like the alternative implementation I had been pushed toward:

```js
for (let sibling = input.nextElementSibling; sibling; sibling = sibling.nextElementSibling) {
```

I wasn't aware of the `except-parens` option! That looks like it'd be perfect for what I was trying to do. Thanks for the pointer.

Reply:
> I really like this rule, it helps catch silly mistakes. What about `except-parens`, which is what it looks like you used?
> 
> ```
>   while ((sibling = sibling.nextElementSibling)) {
> ```

Updated in cc507d08f.